GabijaBern/assignment1/Anagrams.java

```diff
@@ -0,0 +1,85 @@
+//Q1: Given two strings, determine if one is an anagram of the other.
+//Parts: 1)stringSensitive()
+//		 2)stringInsensitive()
+//		 3)sentences()
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class Anagrams {
+
+	
+	//-----------------first part of the question------------------------------
+	public static boolean stringSensitive(String firstWord, String secondWord)
+	{
+		//we could first check if those words are the same length
+		if(firstWord.length()!=secondWord.length()) 
+			return false;
+		
+		Map<Character, Integer> firstMap = createCharMap(firstWord);
+		Map<Character, Integer> secondMap = createCharMap(secondWord);
+			
+		return(firstMap.equals(secondMap));

```

Comment:
Style: Return statement do not usually include `()` around the value (return is a statement, but this makes it look like a function).